clap_mangen/src/render.rs

```diff
@@ -74,18 +72,46 @@ pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
         line.push(roman(" "));
     }
 
-    if cmd.has_subcommands() {
-        let (lhs, rhs) = subcommand_markers(cmd);
-        line.push(roman(lhs));
-        line.push(italic(
-            cmd.get_subcommand_value_name()
-                .unwrap_or_else(|| subcommand_heading(cmd))
-                .to_lowercase(),
-        ));
-        line.push(roman(rhs));
+    line
+}
+
+pub(crate) fn synopsis(roff: &mut Roff, cmd: &clap::Command) {
+    let name = cmd.get_bin_name().unwrap_or_else(|| cmd.get_name());
+    let flatten = cmd.is_flatten_help_set();
+
+    let mut ord_v = Vec::new();
+    if flatten {
+        for subcommand in cmd.get_subcommands() {
+            ord_v.push((
+                subcommand.get_display_order(),
+                format!("{} {}", name, subcommand.get_name().to_owned()),
```

Comment:
Why are we manually building this, rather than using `subcommand.get_bin_name().unwrap_or_else(|| cmd.get_name());`

Reply:
With that change, do we even need a `name` parameter?

Reply:
Cleaned up

Reply:
This change no longer exists, it's marked outdated

Reply:
Whether the lines this originally applied to are there or not, the problem is still there.